src/cim/components/InfraEnv/AgentAlerts.tsx

```diff
@@ -18,22 +26,38 @@ const AgentAlerts: React.FC<AgentAlertsProps> = ({ infraEnv, bareMetalHosts, doc
         h.metadata?.labels?.[INFRAENV_AGENTINSTALL_LABEL_KEY] === infraEnv.metadata?.name,
     );
 
-  return infraBMHs?.some((bmh) => !bmh.status) ? (
-    <Alert
-      title="Metal3 operator is not configured"
-      variant={AlertVariant.warning}
-      isInline
-      className="cim-resource-alerts"
-      actionLinks={
-        <a href={getInfraEnvDocs(docVersion)} target="_blank" rel="noopener noreferrer">
-          Open documentation
-        </a>
-      }
-    >
-      It seems the Metal3 operator is missing configuration which will prevent it to find bare metal
-      hosts in this namespace. Please refer to the documentation for the first time setup steps.
-    </Alert>
-  ) : null;
+  return (
+    <>
+      {infraBMHs?.some((bmh) => !bmh.status) && (
+        <Alert
+          title="Metal3 operator is not configured"
+          variant={AlertVariant.warning}
+          isInline
+          className="cim-resource-alerts"
+          actionLinks={
+            <a href={getInfraEnvDocs(docVersion)} target="_blank" rel="noopener noreferrer">
+              Open documentation
+            </a>
+          }
+        >
+          It seems the Metal3 operator is missing configuration which will prevent it to find bare
+          metal hosts in this namespace. Please refer to the documentation for the first time setup
+          steps.
+        </Alert>
+      )}
+      {aiConfigMap && (
+        <Alert
+          title="Minimum hardware requirements"
+          variant={AlertVariant.info}
+          isInline
+          isExpandable
+          className="cim-resource-alerts"
+        >
+          <HostRequirements {...getHWRequirements(aiConfigMap)} />
```

Comment:
The list in the alert is left-padded too much as seen on the screenshot. That is caused by presence of `<TextContent>` inside `HostRequirements`'s render function.

Since `HostRequirements` component is currently used only on one place, I'd suggest renaming it to `HostRequirementsList` and take the `TextContent` out and use it like this in `MinimalHWRequirements` component:
```
<TextContent>
  <HostRequirementsList {...getHWRequirements(aiConfigMap)} isSNOCluster={isSNOCluster} />
</TextContent>
```

Reply:
good catch. Fixed
![Screenshot from 2022-02-08 15-32-21](https://user-images.githubusercontent.com/2078045/153008098-6a82857b-17cd-4327-b29f-b56f95c31eee.png)